src/NuGetForUnity/Editor/Configuration/NugetConfigFile.cs

```diff
@@ -89,6 +86,29 @@ public class NugetConfigFile
         [NotNull]
         public string RepositoryPath { get; private set; } = Path.GetFullPath(Path.Combine(Application.dataPath, "Packages"));
 
+        /// <summary>
+        ///     Gets or sets the incomplete path that is saved.  The path is expanded and made public via the property above.
+        /// </summary>
+        [NotNull]
+        public string RelativeRepositoryPath
```

Comment:
```suggestion
        internal string ConfiguredRepositoryPath
```
The important part is that this ist the path that is stored inside the `NuGet.config`, it can be absolute / relative / contain environment variables. Eventually there is a even better name.

Reply:
I think the original name "savedRepositoryPath" is not so bad.